src/tasks/platforms/api.ts

```diff
@@ -22,11 +23,15 @@ export class ApiTasks {
     let kube = new KubeHelper(flags)
     return {
       title: 'Verify Kubernetes API',
-      skip: () => flags['skip-kubernetes-health-check'],
       task: async (ctx: any, task: any) => {
         try {
-          await kube.checkKubeApi()
+          cli.info(`› Current Kubernetes context: '${await kube.currentContext()}'`)
```

Comment:
Not all the commands use these API tasks. Is it planned to print the message on a subset of commands only?

Reply:
Basically this task is run only when k8s api is used.